services/github/github-commit-activity.service.js

```diff
@@ -18,11 +23,16 @@ const schema = Joi.object({
   }).required(),
 }).required()
 
+const queryParamSchema = Joi.object({
+  authorFilter: Joi.string(),
+})
+
 export default class GitHubCommitActivity extends GithubAuthV4Service {
   static category = 'activity'
   static route = {
     base: 'github/commit-activity',
     pattern: ':interval(t|y|m|4w|w)/:user/:repo/:branch*',
+    queryParamSchema,
   }
 
   static examples = [
```

Comment:
The example badges are showing "undefined" as the label. Are you able to have a look at this?

Reply:
After years of debugging i figured the issue.
defaultBadgeData is defined *after* examples..., becouse of that, the value of the parent calss is used (BaseService) which is empty object {}.
Moving the static defenition of defaultBadgeData  before examples solves the issue...
But if i change the order eslint is mad at me for breaking the expected order of static properties *sigh*.
Should i change the order in the base service class? is there a reason for this order?

Reply:
Just to avoid this becoming a gigantic refactoring job that changes every service to add one badge, I think the path of least resistance to getting this merged is to change one line in render from

```
const label = interval === 't' ? 'commits' : this.defaultBadgeData.label
```

to

```
const label = interval === 't' ? 'commits' : 'commit activity'
```

We can live with a duplicated string here.

There's some frontend work in progress in https://github.com/badges/shields/pull/9014 which will make a lot of this irrelevant soon anyway.